ts/src/base/Exchange.ts

```diff
@@ -241,6 +241,7 @@ export default class Exchange {
     balance      = {}
     orderbooks   = {}
     tickers      = {}
+    bidsasks     = {}
```

Comment:
should we create a new structure for the bidsasks?

Reply:
yes, i see it's inevitable. no other way, bcz bidsasks seem to be supported on more exchanges and they are separate streams

Reply:
@ttodua can you name a few? (just to get an idea)

Reply:
what i ahve already seen with just quick peek:
binance - https://binance-docs.github.io/apidocs/futures/en/#all-book-tickers-stream
gate
gemini
... would be more